lib/services/summary_service.ex

```diff
@@ -1,18 +1,30 @@
 defmodule Aprb.Service.SummaryService do
   alias Aprb.{Repo, Topic, Summary}
+  import Ecto.Query
 
   def update_summary(topic, event) do
     current_date = Calendar.Date.today! "America/New_York"
     cond do
-      Enum.member?(~w(subscriptions users inquiries purchases conversations), topic) ->
+      Enum.member?(~w(subscriptions users inquiries purchases conversations), topic.name) ->
```

Comment:
we used to pass in topic's name to this method instead of actual `Topic` model, now that we pass the model instead of topic name string, we need to get name from it.